timeless/restaurants/floors/views.py

```diff
@@ -17,11 +17,9 @@
  timeless/views.py::UpdateView, receives data from the first view and
  must save the data to the repository. The tests must include checking if the
  view screens were correctly built and if the data was saved to the repository.
-@todo #95:30min Implement delete floor view. Delete floor view is composed of
- two views: the first view must load the desired floor data onto the screen
- and show an button to delete the selected view, extending
- timeless/views.py::DetailView; second view must show the result message of
- floor deletion.
+@todo #424:30min The last step for deletion is to ask for confirmation before
+ actually deleting the object. When clicking on link on list floors pages a
+ javascript modal should appear asking user for confirmation
```

Comment:
@emilianodellacasa Is this a puzzle that should be in the backend? Besides, I guess it should be a generic solution for all delete operations, not just for floors.

Reply:
@amihaiemil You are right of course, I wanted this to be generic. Let me specify it better